modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path

```diff
@@ -12,6 +12,16 @@
 		<td>//input[@placeholder="Choose an Action"]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ADD_ERROR_MESSAGE_VALIDATION</td>
+		<td>//div[contains(@class,'form-group')]//label[text()='Message']/..//input[@type='text']</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>OBJECT_KEBAB_OPTION</td>
+		<td>//*[contains(text(),'${key_objectLabel}')]/ancestor::div[contains(@class,'dnd-tr')]//*[contains(@aria-controls,'${key_newOption}')]</td>
+		<td></td>
+	</tr>
```

Comment:
We should have existing paths that work, especially the kebab icon. We have multiple generic paths for the kebab icon and we should have an existing path for the input field